Assignment2/ruliii-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,43 @@
+{
+  "@context":{
+    "xsd":"http://www.w3.org/2001/XMLSchema#",
+    "ns": "http://www.ontologies.org/ontologies/Class#",
+    "temperature":"xsd:long",
+    "date":"xsd:dateTime",
+    "name":"xsd:string",
+    "includes":"ns:includes",
+    "hasOwner":"ns:hasOwner",
+        "hasMeasurement":"ns:hasMeasuremet",
+        "hasTemperature":"ns:hasTemperature",
+        "atTime":"ns:atTime"
+     },
+    "@id":"Class01",
+    "@type":"ns:Class",
+  "includes":[
+      {
+          "@id":"Sensor029",
+          "@type":"ns:Sensor",
+          "hasMeasurement":{
+          "@id":"Measurement8401",
+            "@type":"ns:Measurement",
+            "hasTemperature":{
+              "temperature":"29"
+            },
+            "atTime":{
+              "date":"2010-06-12T12:00:12"
+            }
```

Comment:
Both value definitions (hasTemperature and atTime) are wrong. In the first case, you defined the "temperature" as a datatype, thus the correct way is to specify it as "@type":"temperature" and the value as "@value":"29". 
You have to do something similar to redefine the value of the "atTime" attribute.